components/atom/table/src/styles/settings.scss

```
@@ -6,3 +6,4 @@ $bgc-atom-table-row-actionable--hover: $c-gray-light-4 !default;
 $bgc-atom-table-row-actionable--active: $c-gray-light-3 !default;
 $bd-atom-table-row: $bdw-s solid $c-gray-light-5 !default;
 $c-atom-table-header-cell: $c-gray-dark !default;
+$bd-bottom-atom-table-header-cell: 0 !default;
```

```suggestion
$bdb-atom-table-header-cell: 0 !default;
```

following emmet's naming convention... 
https://docs.emmet.io/cheat-sheet/

---

Done, thanks!